assets/dev/js/editor/document/elements/commands-internal/scroll-to-view.js

```diff
@@ -1,6 +1,6 @@
-import CommandInternal from 'elementor-api/modules/command-internal-base';
+import CommandContainerInternal from 'elementor-editor/command-bases/command-container-internal';
```

Comment:
Please use it a 3rd party. Please remove all imports.

Reply:
The fixes are here: https://github.com/elementor/elementor/pull/17728
please merge to [elementor:feature/js-api-commands-refactor](https://github.com/elementor/elementor/tree/feature/js-api-commands-refactor)